tests/test_torchinductor.py

```diff
@@ -2574,6 +2574,20 @@ def run(x):
             expected_kernel = 6
         self.assertEqual(torchinductor.metrics.generated_kernel_count, expected_kernel)
 
+    def test_roll(self):
```

Comment:
It's possible to use to use `torch.testing._internal.common_methods_invocations.sample_inputs_roll` to get the arguments for testing.
Then it would match what's being tested in PyTorch in `test/test_ops.py -k roll` tests.

Reply:
I like this idea.  I created an issue #727